code/game/objects/items/weapons/legcuffs.dm

```diff
@@ -41,7 +42,11 @@
 
 /obj/item/restraints/legcuffs/beartrap/attack_self(mob/user)
 	..()
-	if(ishuman(user) && !user.stat && !user.restrained())
+
+	if(!ishuman(user) || ishuman(user) && user.restrained())
```

Comment:
You can avoid checking for "ishuman()" a second time here.

Reply:
resolved with [7383b94](https://github.com/ParadiseSS13/Paradise/pull/21941/commits/7383b9491f7e56b5739fa51c130edd7c766e0de5)